emukit/examples/preferential_batch_bayesian_optimization/pbbo/util.py

```diff
@@ -0,0 +1,291 @@
+import numpy as np
+from numpy import linalg as la
+import logging
+from typing import Callable, List, Tuple, Dict
+
+'''
+nearestPD and isPD are copied from https://github.com/alan-turing-institute/bocpdms with the following licence:
+
+
+MIT License
```

Comment:
This file should go as well, we cannot add MIT license to emukit at this time

Reply:
Definitely! My bad for not noticing this! Sorry!